openreview/conference/builder.py

```diff
@@ -163,11 +164,20 @@ def __create_decision_stage(self):
         notes = list(self.get_submissions())
         return self.invitation_builder.set_decision_invitation(self, notes)
 
-    def __set_reviewer_reassignment(self, enabled = True):
```

Comment:
why did you remove the underscores here?

Reply:
Because I couldn't call it from set_assignments().  And I expect we'll want to remove the ability to reassign after reviews are in and would want to call it from conference.